package.json

```
@@ -28,7 +28,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
-    "@saleor/macaw-ui": "^0.4.0",
+    "@saleor/macaw-ui": "github:saleor/macaw-ui#9612762",
```

Use the full length hash to make the hash immutable (SHA1 collision)
```suggestion
    "@saleor/macaw-ui": "github:saleor/macaw-ui#961276259adba647e4f8652e5d069765deff8c02",
```